content/sigs/docs/index.adoc

```diff
@@ -0,0 +1,123 @@
+  github: "stackscribe"
+- name: "Zhao Xiaojie (Rick)"
+  id: "surenpi"
+  github: "LinuxSuRen"
+links:
+  gitter: "jenkinsci/docs"
+  googlegroup: "jenkinsci-docs"
+  meetings: "https://docs.google.com/document/d/1uNNo0QJKPHnNp8PGr_jLI8p3K_94ZYD-M0evZOEZ93c/edit#"
+overview: >
+  This special interest group improves Jenkins use and adoption through documentation.
+  The SIG encourages, creates, and reviews documentation contributions and improves documentation with contributors and external communities.
+---
+
+The Documentation special interest group offers a venue for all kinds of documentation creation, improvement, and delivery.
+
+The group focuses on documentation for Jenkins and Jenkins X, including:
+
+* link:/doc/[User documentation] - Using Jenkins
+** link:/doc/tutorials[Tutorials]
+** link:/doc/book/[User Handbook]
+** References like link:/doc/book/pipeline/syntax/[Pipeline Syntax] and link:/doc/pipeline/steps/[Pipeline Steps]
+** How-to Guides
+* link:/doc/developer/[Extension documentation] - Extending Jenkins
+** link:/doc/developer/tutorial/[Tutorials]
+** link:/doc/developer/book/[Developer Reference]
+** link:/doc/developer/guides/[How-To Guides]
+* Solution documentation - Specific Solutions with Jenkins
+** Language solutions like link:/solutions/c/[C/C\+\+], link:/solutions/java/[Java], link:/solutions/php/[PHP], link:/solutions/python/[Python], and link:/solutions/c/[Ruby]
+** Platform solutions like link:/solutions/android/[Android], link:/solutions/docker[Docker], and link:/solutions/embedded[Embedded]
+** Implementation solutions like link:/solutions/pipeline[Pipeline]
+
+Documentation SIG may cooperate with other groups.
+For example, we will be cooperating with the link:/sigs/cloud-native[Cloud-Native Jenkins SIG]
+on topics related to Cloud-Native documentation efforts and
+with the link:/sigs/platform[Platform SIG].
+on platform topics.
+
+=== Topics
+
+* Creating new documentation
+* Improving existing documentation
+* Reviewing documentation contributions
+* Coordinating documentation initiatives
+* Reviewing JEPs related to documentation
+
+=== Ongoing projects
+
+* Identifying newbie-friendly documentation tickets
+* Reviewing Jenkins documentation link:https://github.com/jenkins-infra/jenkins.io/pulls[pull requests]
+* Reviewing Jenkins X documentation link:https://github.com/jenkins-x/jx-docs/pulls[pull requests]
+* link:https://plugins.jenkins.io/[Plugins site] improvements
+
+See the SIG meeting notes for more information about the ongoing projects.
+
+=== Meetings
+
+We have regular meetings on Fridays every two weeks, at *1PM UTC*.
+See the link:/event-calendar/[Jenkins Event Calendar] for the schedule.
+At these meetings we discuss projects and do presentations/demos.
+You can find and contribute to the agenda for the incoming meetings
+link:https://docs.google.com/document/d/1uNNo0QJKPHnNp8PGr_jLI8p3K_94ZYD-M0evZOEZ93c/edit?usp=sharing[here].
+
+Meetings will be conducted and recorded via Zoom.
```

Comment:
```suggestion
Meetings are conducted and recorded using Zoom.
```
Do you want to say where the recordings are available?  I guess one gets the link to join a particular meeting from the Calendar that is referenced above?